rs/backup/BUILD.bazel

```
@@ -43,6 +41,10 @@ rust_library(
     crate_name = "ic_backup",
     proc_macro_deps = MACRO_DEPENDENCIES,
     version = "1.0.2",
+    visibility = [
+        "//rs:system-tests-pkg",
+        "//rs:tools-pkg",
```

I might be wrong but I don't think this target has to be visible to other packages (except the system tests)

```suggestion
        "//rs:system-tests-pkg",
```